step-09/pom.xml

```diff
@@ -74,6 +74,26 @@
             <artifactId>quarkus-websockets-next</artifactId>
         </dependency>
 
+        <!-- Export metrics for OpenTelemetry compatible collectors -->
+        <dependency>
+            <groupId>io.quarkiverse.micrometer.registry</groupId>
+            <artifactId>quarkus-micrometer-registry-otlp</artifactId>
+            <version>3.3.1</version>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-opentelemetry</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.opentelemetry.instrumentation</groupId>
+            <artifactId>opentelemetry-jdbc</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-observability-devservices-lgtm</artifactId>
+            <scope>provided</scope>
+        </dependency>
+
```

Comment:
step 10 includes these dependencies in an Observability section -- need to make sure the placement/section matches with step 10.

Reply:
According to @kdubois review, I will remove this one.